src/main/plugins/org.dita.base/build_preprocess_template.xml

```diff
@@ -454,7 +454,7 @@
         <param name="TABLELINK" expression="${args.tablelink.style}" if="args.tablelink.style" />
         <param name="FIGURELINK" expression="${args.figurelink.style}" if="args.figurelink.style" />
         <param name="ONLYTOPICINMAP" expression="${onlytopic.in.map}" if="onlytopic.in.map"/>
-        <param name="ditaot.default.language" expression="${default.language}"/>
+        <param name="defaultLanguage" expression="${default.language}"/>
```

Comment:
For that program they're all upper case, but there is no one convention across all of our XSLT. Most of the other programs use a mix, with camelCase quite common. Whatever name is used, it has to be the same in all cases where we use it.